### PR TITLE
Create readme in examples directory

### PR DESCRIPTION
**Pull request by najamelan**
I will write Nathan to ask them if this pull request is ok with them. Please don't merge before confirmation.

### CODE REVIEW

**Comment by najamelan** [comment]
Nathan has agreed to being linked!

File: examples/README.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Actix Examples
+
+The following examples complete the documentation and can help you understand how to use Actix:
```
Comment (DoumanAsh) [inline examples/README.md]: >The following examples complete the documentation

By `complete` did you mean complement?

**Comment by najamelan** [comment]
@DoumanAsh sorry, I didn't realize I could merge it myself. nothing to review...

**Comment by DoumanAsh** [comment]
Sorry, about not re-reviewing, sometimes my gtihub notifications are messed.
But well my comment was minor so you can merge as it is

**Comment by DoumanAsh** [comment]
@najamelan But please next time be sure to squash commits before merging or use merge and squash in github UI

**Pull request merged**